app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  has_many :memberships, dependent: :destroy
```

Comment:
I think we should probably add a uniqueness validation on the user model and at the database level to avoid problems caused by duplicate records.

Reply:
yes, will do

Reply:
I also added presence and uniqueness to auth_id